setup.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+from setuptools import find_packages, setup
+
+setup(
+    name="musekafka",
+    use_scm_version={"write_to": "src/musekafka/_version.py"},
+    setup_requires=["setuptools_scm"],
+    description="Python client to simplify work with Kafka consumers and producers.",
+    long_description=open("README.md").read(),
+    long_description_content_type="text/markdown",
+    license="MIT",
```

Comment:
I thought we used `BSD 3`?

Reply:
Not that I'm aware of. Who stated that?

Reply:
here is a DM I had with Yusuf awhile back

```
(Also totally unrelated but if I created a Dailymuse OSS lib, what sort of license would I need?)

license is BSD 3-clause: https://github.com/dailymuse/oz/blob/develop/LICENSE
GitHubGitHub
dailymuse/oz
oz - Middleware and scaffolding for tornado-based webapps
5:31
just update the copyright when you import that :slightly_smiling_face:

dcad:spiral_calendar_pad:  5:31 PM
Awesome! Thanks!
```

Oz uses the BS3-3 as well. idk if this is just an old policy but thought I'd mention.

Reply:
The main difference between the licenses being that

```
but with a 3rd clause that prohibits others from using the name of the project or its contributors to promote derived products without written consent.
```

Which I think we would want that kind of protection right?